src/spec/client/services/product-projections.spec.coffee

```diff
@@ -98,10 +110,11 @@ describe 'ProductProjectionService', ->
       .filter('foo:bar')
       .filterByQuery('foo:bar')
       .filterByFacets('foo:bar')
+      .markMatchingVariants(true)
```

Comment:
I just wanted to point out that only when you do ```asSearch``` only than it makes sense to set markMatchingVariants.